### PR TITLE
React-Todo

### PR DESCRIPTION
**Pull request by bcuz**

### CODE REVIEW

**Comment by bcuz** [comment]
deployed to github pages here: https://bcuz.github.io/React-Todo/

File: src/App.js [hunk 1]
```diff
@@ -28,17 +28,18 @@ class App extends React.Component {
   handleSubmit = (e) => {
     e.preventDefault()
```
Comment (jeffyjkang) [inline src/App.js:30]: nicely done catching edge cases of attempting an empty input, another thing you can implement, in React is if you put just the word 'required' as a field in your input, it makes a little modal pop up

File: src/App.js [hunk 2]
```diff
@@ -35,10 +35,12 @@ class App extends React.Component {
         completed: false
       }
```
Comment (jeffyjkang) [inline src/App.js:36]: great work on the use of prevState

**Review by jeffyjkang — APPROVED** [review]
nicely done MVP

**Pull request merged**